src/components/AppWrapper.jsx

```diff
@@ -18,6 +18,7 @@ import { usePreferedTheme } from 'hooks/usePreferedTheme'
 
 import schema from '../schema'
 import { ConditionalWrapper } from './ConditionalWrapper'
+import { CustomWallPaperProvider } from '../hooks/useCustomWallpaperContext'
```

Comment:
nit: We can use an alias here : 
`import { useCustomWallpaperContext } from 'hooks/useCustomWallpaperContext'`